requirements.txt

```diff
@@ -0,0 +1,7 @@
+urllib3 >= 1.25.8
+six >= 1.14.0
+certifi==2019.11.28
```

Comment:
Those requirements seem very strict, we'll be forcing them on all users of this library. Is there a way to make them more flexible?